.travis.yml

```diff
@@ -10,6 +10,9 @@ env:
   - MYUSEMC=true MYPYTHON_VERSION=3.5
   - MYUSEMC=true MYPYTHON_VERSION=3.6
   - MYUSEMC=true MYPYTHON_VERSION=3.7
```

Comment:
Following NEP 29 (https://numpy.org/neps/nep-0029-deprecation_policy.html) I suggest we drop support for 3.7 and earlier.

Reply:
I suggest remove 3.5 and 3.6, but keep 3.7, since many of our diffpy software replies on 3.7 and earlier at this moment.

Reply:
@sbillinge How do you think?